src/linear_algebra/alternating.lean

```diff
@@ -144,6 +144,27 @@ f.to_multilinear_map.map_smul' v i r x
   f v = 0 :=
 f.map_eq_zero_of_eq' v i j h hij
 
+lemma map_coord_zero {m : ι → M} (i : ι) (h : m i = 0) : f m = 0 :=
+f.to_multilinear_map.map_coord_zero i h
+
+@[simp] lemma map_update_zero (m : ι → M) (i : ι) : f (update m i 0) = 0 :=
+f.to_multilinear_map.map_update_zero m i
+
+@[simp] lemma map_zero [nonempty ι] : f 0 = 0 :=
+f.to_multilinear_map.map_zero
+
+/-- Make a copy of `m` that when applied is defeq to `f`. -/
+@[simps]
+def copy (m : alternating_map R M N ι) (f : (ι → M) → N) (hf : ⇑m = f) : alternating_map R M N ι :=
```

Comment:
Hmm, this is an interesting technique! I don't see yet why we should care (much) about defeq here, why not define a new `_apply` lemma instead?

Reply:
Because I need the map to be defeq in order to use `some_det_copy.map_etc` as the proof of `det_etc`.

We have this technique already for bundled subgroups like [`subgroup.copy`](https://leanprover-community.github.io/mathlib_docs/group_theory/subgroup.html#subgroup.copy), although I don't think we have it for any maps.